crates/chronicle/src/bootstrap/cli.rs

```diff
@@ -1056,6 +1056,33 @@ impl SubCommand for CliModel {
                         .value_parser(["data", "graphql"])
                         .default_values(&["data", "graphql"])
                         .help("which API endpoints to offer")
+                    )
+                    .arg(
+                        Arg::new("load-opa-from-url")
+                        .long("load-opa-from-url")
+                        .required(false)
+                        .takes_value(true)
+                        .value_hint(ValueHint::Url)
+                        .help("URL for loading OPA policy bundle")
+                    )
+                    .arg(
+                        Arg::with_name("opa-policy-name")
+                            .long("opa-policy-name")
+                            .help("Name of the OPA policy to use if 'load-opa-from-url' is selected")
+                            .takes_value(true)
+                            .requires("load-opa-from-url"),
+                    )
+                    .arg(
+                        Arg::with_name("opa-policy-entrypoint")
+                            .long("opa-policy-entrypoint")
+                            .help("Entrypoint to the OPA policy if 'load-opa-from-url' is selected")
+                            .takes_value(true)
+                            .requires("load-opa-from-url"),
+                    )
+                    .group(
+                        ArgGroup::with_name("opa-policy-from-url-args")
+                            .args(&["opa-policy-name", "opa-policy-entrypoint"])
+                            .requires("load-opa-from-url"),
```

Comment:
Curious what this adds given that the arguments separately also require it.

Reply:
Think I've fixed this. Thanks